qiskit/synthesis/linear_phase/cz_depth_lnn.py

```diff
@@ -133,20 +133,22 @@ def _create_patterns(n):
 
 
 def synth_cz_depth_line_mr(mat: np.ndarray):
-    """Synthesis of a CZ circuit for linear nearest neighbour (LNN) connectivity,
+    r"""Synthesis of a CZ circuit for linear nearest neighbour (LNN) connectivity,
     based on Maslov and Roetteler.
 
     Note that this method *reverts* the order of qubits in the circuit,
-    and returns a circuit containing CX and phase (S, Sdg or Z) gates.
+    and returns a circuit containing :class:`.CXGate`\s and phase gates
+    (:class:`.SGate`, :class:`.SdgGate` or :class:`.ZGate`).
 
     Args:
         mat: an upper-diagonal matrix representing the CZ circuit.
-            mat[i][j]=1 for i<j represents a CZ(i,j) gate
+            ``mat[i][j]=1 for i<j`` represents a ``cz(i,j)`` gate
 
     Return:
-        QuantumCircuit: a circuit implementation of the CZ circuit of depth 2*n+2 for LNN connectivity.
+        QuantumCircuit: a circuit implementation of the CZ circuit of depth ``2*n+2`` for LNN
```

Comment:
Should this be math mode for consistency with the other changes?
```suggestion
        QuantumCircuit: a circuit implementation of the CZ circuit of depth :math:`2n+2` for LNN
```